tools/release_builder.py

```diff
@@ -15,6 +14,7 @@
 import urllib.request
 import zipfile
 from collections import OrderedDict
+import http.client
```

Comment:
There's an import sort tool for python isort which is reasonable if you're actually developing in python.  I'm hesitant to spend time on that level of tooling for our use patterns.